main.go

```diff
@@ -46,10 +46,7 @@ func init() {
 	utilruntime.Must(corev1.AddToScheme(scheme))
 	utilruntime.Must(appsv1.AddToScheme(scheme))
 	utilruntime.Must(rbac.AddToScheme(scheme))
-
-	if infrastructure.Current.Type == infrastructure.OpenShift {
-		utilruntime.Must(routev1.AddToScheme(scheme))
-	}
+	utilruntime.Must(routev1.AddToScheme(scheme))
```

Comment:
why remove `if openshift`? would it work on k8s ?

Reply:
Yes, `AddToScheme()` merely registers the types for serde as far as I understand. It definitely works on kubernetes like this.